virtualizarr/xarray.py

```diff
@@ -27,6 +28,7 @@ def open_virtual_dataset(
     loadable_variables: Optional[Iterable[str]] = None,
     indexes: Optional[Mapping[str, Index]] = None,
     virtual_array_class=ManifestArray,
+    reader_options: Optional[dict] = {'storage_options':{'key':'', 'secret':'', 'anon':True}},
```

Comment:
When you normally point `xr.open_dataset` at an S3 url, you don't need to pass `reader_options` do you? Can we try to follow the signature of `xr.open_dataset` as closely as possible? (Maybe this already is as close as we can get)

Reply:
I could be wrong, but I thought you had to pass in some sort of fsspec/s3fs mapper. 

For me this fails:
``` import xarray as xr
ds = xr.open_dataset('s3://carbonplan-share/virtualizarr/local.nc')
```